src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -0,0 +1,322 @@
+import React, { useState } from 'react';
+import { ArrayField, useFieldArray, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+import {
+  DatePeriod,
+  DatePeriodOptions,
+  FormWeekdays,
+  Language,
+  ResourceState,
+  ResourceStateApiOption,
+  ResourceStateOption,
+  TimeSpan as TimeSpanApiFormat,
+  TimeSpanFormFormat,
+} from '../../common/lib/types';
+import { transformToApiFormat } from '../../common/utils/date-time/format';
+import Datepicker from '../../components/datepicker/Datepicker';
+import { PrimaryButton, SecondaryButton } from '../../components/button/Button';
+import { ErrorToast, SuccessToast } from '../../components/notification/Toast';
+import OpeningPeriodDescription from '../description/OpeningPeriodDescription';
+import TimeSpan from '../time-span/TimeSpan';
+import './OpeningPeriodForm.scss';
+
+type OpeningPeriodDataBase = {
```

Comment:
I wonder if OpeningPeriodBaseData would be better? DataBase feels like it's little too close to a word database, which has little different meaning I think 😁

Reply:
Yes, good idea 😁

Reply:
This was completely redundant type declaration.

Reply:
Removed declaration